src/applications/vaos/services/appointment/transformers.v2.js

```diff
@@ -165,6 +165,21 @@ export function isPastAppointment(appt) {
   return apptDateTime.add(threshold, 'minutes').isBefore(moment());
 }
 
+/**
+ *  Determines whether current time is before appointment time
+ * @param {*} appt VAOS Service appointment object
+ * @param {*} isRequest is appointment a request
+ */
+export function isUpcomingAppointment(appt, isRequest) {
+  const apptDateTime = moment(appt.start);
+  return (
+    !isRequest &&
+    !isPastAppointment(appt) &&
+    apptDateTime.isValid() &&
+    apptDateTime.isAfter(moment().startOf('day'))
+  );
+}
+
```

Comment:
I would consider a different function name based on the description of what it does and the fact that there is another function in the code base with the same name.

../src/applications/vaos/services/appointment/index.js